src/components/Drawer.js

```diff
@@ -0,0 +1,146 @@
+const React = require('react');
+const Velocity = require('velocity-animate');
+
+const StyleConstants = require('../constants/Style');
+
+const Icon = require('../components/Icon');
+
+const Drawer = React.createClass({
+  propTypes: {
+    duration: React.PropTypes.number,
+    easing: React.PropTypes.array,
+    isOpen: React.PropTypes.bool,
```

Comment:
We've decided to drop the `isOpen` prop on modals and modal like components in favor of letting the parent decide when to render.  Using a prop like this prevents the component from unmounting when `isOpen` is false.